zzzz_modular_occulus/code/modules/mob/living/simple_animal/hostile/siren/replicant.dm

```diff
@@ -0,0 +1,298 @@
+/mob/living/simple_animal/hostile
+	var/idle_vision_range = 9 //If a mob is just idling around, it's vision range is limited to this. Defaults to 9 to keep in line with original simple mob aggro radius
+	var/ranged_cooldown_cap = 3 //What ranged attacks, after being used are set to, to go back on cooldown, defaults to 3 life() ticks
+	var/retreat_distance = null //If our mob runs from players when they're too close, set in tile distance. By default, mobs do not retreat.
+	var/search_objects = 0 //If we want to consider objects when searching around, set this to 1. If you want to search for objects while also ignoring mobs until hurt, set it to 2. To completely ignore mobs, even when attacked, set it to 3
+	var/ranged_cooldown_time = 150
+/mob/living/simple_animal/hostile/siren/proc/GiveTarget(var/new_target) //Step 4, give us our selected target
+	..()
```

Comment:
As our actions pointed out, there is no need to call ..() on a proc with no parent.